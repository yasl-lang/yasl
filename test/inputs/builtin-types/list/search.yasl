let x = [1, 2, 3, 1]
echo x->search(4)
echo x->search(3)
echo x->search(1)

# Search starting from index 2
echo x->search(1, 2)

let y = ['a', 'b', 'c', 'a']
echo y->has('sadasda')
echo y->has('c')
echo y->has('a')

# Search starting from index 2
echo y->has('a', 2)
echo y->has('b', 2)
