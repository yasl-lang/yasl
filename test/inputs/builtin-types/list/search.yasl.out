undef
2
0
3
false
true
true
true
false
